src/Compilers/Core/Portable/Symbols/Attributes/CommonParameterWellKnownAttributeData.cs

```diff
@@ -109,5 +109,37 @@ public bool HasIUnknownConstantAttribute
             }
         }
         #endregion
+
+        private bool _hasNotNullWhenFalseAttribute;
+        public bool HasNotNullWhenFalseAttribute
```

Comment:
>public bool HasNotNullWhenFalseAttribute [](start = 8, length = 40)

I assume VB doesn't take advantage of this member, please move it to C# specific type. Declare one if it doesn't exist.